A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg

```diff
@@ -151,7 +151,7 @@ wing_dihedral = 7 ; Dihedral angle Lambda (DEGREES)
 wing_incidence = 0 ; Wing incidence (DEGREES)
 wing_twist = -2 ; Wing twist epsilon (DEGREES)
 oswald_efficiency_factor = 0.75 ; Wing Oswald efficiency factor e (non dimensional)
-wing_winglets_flag = 0 ; Has winglets true/false
```

Comment:
This should probably be it's own issue. We should really investigate the effect changes to the flight model have extensively before deciding to adopt the change.

Reply:
Good point. I’ll revert it here and put it in a separate pull for further testing.

Reply:
Done @UncleClapton :)